presto-hive-hadoop2/src/test/java/io/prestosql/plugin/hive/TestHiveHadoop2Plugin.java

```diff
@@ -59,4 +59,21 @@ public void testGcsAccessTokenAndHiveCachingMutuallyExclusive()
                     .shutdown();
         }).hasMessageContaining("Use of GCS access token is not compatible with Hive caching");
     }
+
+    @Test
+    public void testRubixCache()
```

Comment:
Can we have some assertion here?

Reply:
> Can we have some assertion here?

Not failing and producing `ConnectorFactory` factory means Guice initialized successfully.